### ISSUE TITLE
Parsedown setSafeMode True or False?

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#82]
In GitLab by @moiri on Oct 4, 2018, 09:55

It was mentioned that it might be useful to allow HTML within markdown.

Given that only Researchers are entering markdown texts it could be an option to disable the safe mode.

This can be done like so:
```diff
--- a/server/page/BasePage.php
+++ b/server/page/BasePage.php
@@ -139,7 +139,7 @@ abstract class BasePage
             // User input handler
             "user_input" => new UserInput($db),
         );
-        $this->services['parsedown']->setSafeMode(true);
+        $this->services['parsedown']->setSafeMode(false);
         $this->fetch_page_info($keyword);
         if($this->id_navigation_section != null)
             $this->services['nav'] = new Navigation($router, $db, $keyword,

```

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Oct 17, 2018, 14:55

In order to increase the flexibility for designing input it was decided to disable the safe mode.

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Oct 19, 2018, 09:53

closed via commit c0657ea6fa86cf718831d4d93be8348fa58cc1bd
